docs/appendix/server-component-manifest.md

```diff
@@ -27,6 +27,14 @@ The manifest file contains the following properties:
 - **`license`**
 
   (Optional but recommended) Defines the license code of the component. For example, Zowe core components have `EPL-2.0` value in this field.
+  
+- **`schemas`**
```

Comment:
I'm trying to think of the case if this should be plural or singular. Any examples we can think of other schemas could come in? But plural could be the safe case.

Reply:
I don't know yet, but I'm making it an object for future-proofing.